tests/kernel/qa_test_klp.pm

```diff
@@ -29,7 +30,7 @@ sub run {
     my ($test_type) = $git_repo =~ /qa_test_(\w+).git/;
 
     (is_sle(">12-sp1") || !is_sle) ? $self->select_serial_terminal() : select_console('root-console');
-    zypper_call('ar -f -G ' . get_required_var('QA_HEAD_REPO') . ' qa_head');
+    add_qa_head_repo;
```

Comment:
Again, the same question:
Regards to `zypper_ar` used in `add_qa_head_repo()` is `zypper_call("--gpg-auto-import-keys ref --repo $name"` sufficient instead of `-G -f` parameters? If we don't have good keys, maybe `zypper_ar` should be changed to use `zypper_call("ar -G -f -p $prio $url $name").